### PR TITLE
adding support for Image

### PR DESCRIPTION
**Pull request by HemaZ**
Hi, 
I've added support for publishing images.

### CODE REVIEW

**Comment by HemaZ** [comment]
@wolfv could you check this PR. if you need any modification please tell me. 
Thanks for your great efforts.

File: jupyros/ros_widgets.py [hunk 1]
```diff
@@ -3,7 +3,13 @@
 except:
     print("The rospy package is not found in your $PYTHONPATH. Subscribe and publish are not going to work.")
     print("Do you need to activate your ROS environment?")
-
+try:
+    from cv_bridge import CvBridge, CvBridgeError
+    import cv2
+    bridge = CvBridge()
+except:
+    print("CV Brdige is not installed, please install it to publish Images ")
```
Comment (wolfv) [inline jupyros/ros_widgets.py]: There is a typo here (Brdige)

File: jupyros/ros_widgets.py [hunk 2]
```diff
@@ -21,6 +27,12 @@ def add_widgets(msg_instance, widget_dict, widget_list, prefix=''):
     """
     # import only here so non ros env doesn't block installation
     from genpy import Message
+    if msg_instance._type.split('/')[-1] == 'Image':
+        w = widgets.Text()
+        widget_dict['img'] = w
+        w_box = widgets.HBox([widgets.Label(value='image path:'), w])
```
Comment (wolfv) [inline jupyros/ros_widgets.py]: Maybe uppercase `Image` in the value field?

**Comment by HemaZ** [comment]
@wolfv  Updated. 
Thank You

**Pull request merged**